Fix example R calculation in edge documentation

xmatthias opened the pull request:
## Summary
Fix example R calculation in edge documentation (as pointed out on slack, the old calculation has at least one typo in the "number example")

We should not calculate with 100% loss. However, the old formulae seems incorrect in total.

@silvavn i would appreciate if you could have a look at this.
Adding this example was your work initially (#3745) - so maybe i'm completely wrong on this and did not understand something...

The formulas used now are from [this page](https://analyzingalpha.com/risk-reward-ratio-for-stocks#:~:text=The%20risk%2Dreward%20ratio%20measures,risking%20%241%20to%20make%20%242.) - but other pages seem to use the same formula.

## Whats new
To have a reference on the changes (since this is hard to read in diffs, and needs rendering):

![2020-10-08-102441_779x629_scrot](https://user-images.githubusercontent.com/5024695/95433686-92ceeb80-0950-11eb-8ce6-371f296829ee.png)



Old version:
![2020-10-08-081320_769x437_scrot](https://user-images.githubusercontent.com/5024695/95421654-23e89700-093e-11eb-8b0e-81baadead1d1.png)

silvavn commented:
Seems correct. Small fix is that the potential profit is of $ 3.33 to $1 (not 3 to 1). Also, its confusing when the mash shows 10-(10*(1-0.15)) when it just says stop_loss. A more straightforward presentation or explanation would be nice.

xmatthias commented:
Good point! I've fixed this now (and updated the image in the first post)

silvavn commented:
On a second thought... This is wrong...  Your potential loss can't be larger than your investment (unless you are leveraged). In the first image the potential loss is of 150, when the investment is 100. Instead of using investment it should be really be share count...
See this: https://www.investopedia.com/terms/r/riskrewardratio.asp AND https://www.investopedia.com/articles/stocks/11/calculating-risk-reward.asp

xmatthias commented:
Great catch! 

I've adjusted it now mostly back to how it was.
Just added the loss side - as i don't think a loss of 100% is realistic given we recommend the use of stoploss.
